addons/account_peppol/models/account_edi_proxy_user.py

```diff
@@ -110,19 +110,16 @@ def _get_proxy_identification(self, company, proxy_type):
             return f'{company.peppol_eas}:{company.peppol_endpoint}'
         return super()._get_proxy_identification(company, proxy_type)
 
-    def _peppol_import_invoice(self, attachment, partner_endpoint, peppol_state, uuid):
+    def _peppol_import_invoice(self, attachment, partner_endpoint, peppol_state, uuid, journal):
```

Comment:
Move the whole method on the journal instead? It seems more logical to me and I would like one day to remove the dependency with the proxy user